src/bin/winkernel/nmrprovider.c

```diff
@@ -0,0 +1,109 @@
+#include "quic_platform.h"
+#include <wdm.h>
+#include "msquic.h"
+#include "quic_trace.h"
```

Comment:
Should we add some logs to this file to understand when things connect/disconnect?

Reply:
Added